tests/IceRpc.Protobuf.Tests/OperationTests.cs

```diff
@@ -21,12 +21,142 @@ public void Unary_rpc_with_empty_param_and_return()
         Assert.That(async () => await client.UnaryOpWithEmptyParamAndReturnAsync(new Empty()), Throws.Nothing);
     }
 
+    [Test]
+    public void Client_streaming_rpc()
+    {
+        // Arrange
+        var service = new MyOperationsService();
+        var invoker = new ColocInvoker(service);
+        var client = new MyOperationsClient(invoker);
+
+        // Act/Assert
+        Assert.That(async () => await client.ClientStreamingOpAsync(GetDataAsync()), Throws.Nothing);
+        Assert.That(service.Empties, Has.Count.EqualTo(3));
+
+        static async IAsyncEnumerable<Empty> GetDataAsync()
+        {
+            await Task.Yield();
+            yield return new Empty();
```

Comment:
While I have nothing against Empty messages, it would be nice to also test "real" messages,

Reply:
I will fix this along https://github.com/icerpc/icerpc-csharp/issues/3708